### ISSUE TITLE
Two Column Mode for iPad and Desktop

### ISSUE THREAD

**Issue by rahulakrishna** [aeharding/voyager#735]
Christian was working on a 2 column mode for iPad and has shared some mockups on Mastodon. It looks good and seems like a good idea to implement it in our app

![image](https://github.com/aeharding/voyager/assets/10240002/b1ab1b94-4a91-4ee2-9a5e-960863d7b4a6)

**Comment by aeharding** [comment]
Hey, I was looking into this a while back! It's actually totally doable, just a bunch of edge cases to consider.

**Comment by boozook** [comment]
I agree with two-column layout on iPad, __but only for fullscreen window mode__, not for split or slide-over.
